core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -1815,6 +1816,14 @@ public ServerBuilder disableDateHeader() {
         enableDateHeader = false;
         return this;
     }
+    /**
+     * Sets the requestTarget not to decode encoded {@code "requestTarget"}.
+     */
+
+    public ServerBuilder percentEncodingPreserved() {
```

Comment:
Sorry about the shift in direction, but I propose the following [direction](https://github.com/line/armeria/issues/5219#issuecomment-1886185265) instead (which probably makes this method obsolete).
Can you check it and see if it makes sense?

Reply:
Thanks for advice! I didn't think about each server/client.
I understand what I to do.
I'll fix PR ASAP.

Reply:
@HongGeonUi Gentle ping :wink:

Reply:
@HongGeonUi Any updates?